### PR TITLE
fix(systemtags): followup #48786; `Types::STRING` and note`min-height`

### PR DESCRIPTION
**Pull request by skjnldsv**
Followup https://github.com/nextcloud/server/pull/48786 with two tiny fixes

- fix(core): use `Types::STRING` in etag systemtag migration
- fix(systemtags): set min height on picker note

### CODE REVIEW

**Review by provokateurin — APPROVED** [review]

File: apps/systemtags/src/components/SystemTagPicker.vue [hunk 1]
```diff
@@ -484,6 +484,13 @@ export default defineComponent({
 	bottom: 0;
 	padding-block: 8px;
 
+	:deep(.notecard) {
+		// min 2 lines of text to avoid jumping
+		min-height: calc(1.5em * 2);
+		line-height: 1.5em;
```
Comment (susnux) [inline apps/systemtags/src/components/SystemTagPicker.vue:490]:
```suggestion
		min-height: 2lh;
		line-height: 1.5;
```

Comment (susnux) [reply apps/systemtags/src/components/SystemTagPicker.vue:490]: `lh` = line height :)

Comment (skjnldsv) [reply apps/systemtags/src/components/SystemTagPicker.vue:490]: TIL, thanks :)

Comment (skjnldsv) [reply apps/systemtags/src/components/SystemTagPicker.vue:490]: Probably don' even need to override the line-height property then

Comment (skjnldsv) [reply apps/systemtags/src/components/SystemTagPicker.vue:490]: Done

**Review by susnux — APPROVED** [review]

**Pull request merged**